apps/admin/frontend/src/screens/hardware_diagnostics_screen.tsx

```diff
@@ -67,6 +84,17 @@ export function HardwareDiagnosticsScreen(): JSX.Element {
             : 'External Power Supply'
           : '-'}
       </P>
+      <H6 as="h3">Memory</H6>
```

Comment:
Not sure how regular users think about this, but "Memory" could be interpreted as RAM.
```suggestion
      <H6 as="h3">Storage</H6>
```